.github/workflows/ci-testing-deploy.yml

```diff
@@ -894,6 +899,7 @@ jobs:
         run: ./ci/github/unit-testing/aws-library.bash install
       - name: typecheck
         run: ./ci/github/unit-testing/aws-library.bash typecheck
```

Comment:
Hmm I can see that some of the new services such as resource-usage-tracker, efs-guardian, cluster-keeper are missing here, I guess we should add them here?

Reply:
I am not a big fan of these bash scripts. I think they are a lot of duplicated code. Probably we should think if it is not possible to have only one script with arguments. Also these scripts usually tend to be slightly different than the commands we execute locally.